edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -85,12 +85,12 @@ var edx = edx || {};
                               action: 'stop'
                             },
                             success: function() {
-                              // Reloading page will reflect the new state of the attempt
-                              location.reload();
+                              // change the location of the page to the active exam page
+                              // which will reflect the new state of the attempt
+                              location.href = self.model.get('exam_url_path');
```

Comment:
Hi @muhhshoaib just confirming will this force reload the browser (across all browser types?) if the user is already on the same page as 'exam_url_path'?